client/asset/btc/btc.go

```diff
@@ -407,12 +407,14 @@ func (btc *ExchangeWallet) Fund(value uint64, nfo *dex.Asset) (asset.Coins, erro
 	if value == 0 {
 		return nil, fmt.Errorf("cannot fund value = 0")
 	}
-	utxos, _, _, unconf, err := btc.spendableUTXOs(nfo.FundConf)
+	// Now that we allow funding with 0 conf UTXOs, some more logic could be
+	// used out of caution, including preference for >0 confs.
+	utxos, _, avail, _, err := btc.spendableUTXOs(0)
```

Comment:
Another list of todos for now or later.

1. Can drop the argument to `spendableUTXOS`.
2. Can drop the `isDexChange` function, the `tradeChange` map, and its mutex. 
3. Can drop arguments to `Balance` and simplify `balance` logic (related to `BalanceSet` changes).

Reply:
Good point about removing the call to `isDexChange` from `spendableUTXOs`, but I suspect `isDexChange` might be valuable when prioritizing utxo selection.

Reply:
But I'm inclined to keep the args for `fund` and `spendableUTXOs` even if we don't use it right now.  I can imagine a client configuration setting that lets the user decide if they want to use unconfirmed outputs or not to create the split txns that fund the orders.